dpgen/dispatcher/LSF.py

```diff
@@ -102,14 +102,19 @@ def sub_script_head(self, res):
         ret = ''
         ret += "#!/bin/bash -l\n#BSUB -e %J.err\n#BSUB -o %J.out\n"
         if res['numb_gpu'] == 0:
-            ret += '#BSUB -R span[ptile=%d]\n#BSUB -n %d\n' % (
-                res['node_cpu'], res['numb_node'] * res['task_per_node'])
+            ret += '#BSUB -n %d\n#BSUB -R span[ptile=%d]\n' % (
+                res['numb_node'] * res['task_per_node'], res['node_cpu'])
         else:
             if res['node_cpu']:
                 ret += '#BSUB -R span[ptile=%d]\n' % res['node_cpu']
-                # It is selected only for the situation that GPU is related to CPU node.
-            ret += '#BSUB -R "select[ngpus >0] rusage[ngpus_excl_p=1]"\n#BSUB -n %d\n' % (
-                res['numb_gpu'])
+            if 'new_lsf_gpu' in res and res['new_lsf_gpu'] == True:
```

Comment:
```suggestion
            if res.get('new_lsf_gpu', False):
```

Reply:
Thanks, it will be modified in the next commit.